### PR TITLE
translate guri,pst packet and update lev packet

### PR DESCRIPTION
**Pull request by earthnash**

### CODE REVIEW

**Comment by MasterDomino** [comment]
@earthnash We need testing.

File: OpenNos.GameObject/Character.cs [hunk 1]
```diff
@@ -2161,7 +2161,7 @@ public string GenerateLev()
             {
                 specialist = Inventory.LoadBySlotAndType<SpecialistInstance>((byte)EquipmentType.Sp, InventoryType.Wear);
             }
-            return $"lev {Level} {LevelXp} {(!UseSp || specialist == null ? JobLevel : specialist.SpLevel)} {(!UseSp || specialist == null ? JobLevelXp : specialist.XP)} {XPLoad()} {(!UseSp || specialist == null ? JobXPLoad() : SPXPLoad())} {Reput} {GetCP()} {HeroXp} {HeroLevel} {HeroXPLoad()}";
+            return $"lev {Level} {LevelXp} {(!UseSp || specialist == null ? JobLevel : specialist.SpLevel)} {(!UseSp || specialist == null ? JobLevelXp : specialist.XP)} {XPLoad()} {(!UseSp || specialist == null ? JobXPLoad() : SPXPLoad())} {Reput} {GetCP()} {HeroXp} {HeroLevel} {HeroXPLoad()} {0}";
```
Comment (Ciapa) [inline OpenNos.GameObject/Character.cs]: Please explain this change in detail, it doesnt change anything in the Game.

Comment (earthnash) [inline OpenNos.GameObject/Character.cs]: In TH client if we don't have the last parameter(Should I use this word?idk sorry 🤤),the job level and hero level won't  show.

**Comment by earthnash** [comment]
sorry for my misspelling.it should have been "dumb."

**Comment by MasterDomino** [comment]
YOLO, might be unstable also .ToString() on string properties FUCK YEAH :D

**Pull request merged**